app/controllers/application_controller.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  def index
+    render 'application/index'
```

Comment:
I believe that this can be removed.
If you have `view` in expected path, in this case `application/index`,
you don't have to specify `render` option.

See https://guides.rubyonrails.org/layouts_and_rendering.html#using-render